climakitae/explore/vulnerability.py

```diff
@@ -40,11 +42,49 @@ def metric_agg(da, metric, threshold):
         "median": np.median,
```

Comment:
Don't need mean or median options here. All of the designated metrics are high/low or min/max. Focus is on extremes.

Reply:
Same in Ln 125